honeybee_vtk/write.py

```diff
@@ -14,21 +14,21 @@
 
 
 def write_vtk(file_path, *, file_name=None, include_grids=True,
-                include_vectors=True, writer='vtk'):
+              include_vectors=True, writer='vtk'):
```

Comment:
I could not align like you have done here. How did you manage to do? With spacing?

Reply:
manually! :)